terraform/main.tf

```
@@ -181,6 +181,40 @@ module "grants_prepared_data_bucket" {
   ]
 }
 
+module "ffis_source_data_bucket" {
+  source  = "cloudposse/s3-bucket/aws"
+  version = "3.0.0"
+  context = module.s3_label.context
+  name    = "ffis_source_data"
+
+  acl                          = "private"
+  versioning_enabled           = true
+  sse_algorithm                = "AES256"
+  allow_ssl_requests_only      = true
+  allow_encrypted_uploads_only = false
+  source_policy_documents      = [data.aws_iam_policy_document.ses_source_data_s3_access.json]
+
+  lifecycle_configuration_rules = [
+    {
+      enabled                                = true
+      id                                     = "rule-1"
+      filter_and                             = null
+      abort_incomplete_multipart_upload_days = 1
+      transition                             = [{ days = null }]
+      expiration                             = { days = null }
+      noncurrent_version_transition = [
+        {
+          days          = 30
+          storage_class = "GLACIER"
+        },
+      ]
+      noncurrent_version_expiration = {
+        days = 2557 # 7 years (includes 2 leap days)
+      }
```

I know this PR isn't intended to fully implement #61, but I think it'd be worth implementing the rules around object expiration here. 
```suggestion
      expiration                    = { days = 30 }
      noncurrent_version_transition = [] # This might need to be [{ days = null }] ?
      noncurrent_version_expiration = { days = null }
```

---

yup! I made those adjustments with my latest commit